packages/core/echo/echo-typegen/test/proto/test.proto

```diff
@@ -23,7 +23,7 @@ message Contact {
 
   option (object) = true;
 
-  string name = 1;
+  string name = 1 [primary = true, required = true];
```

Comment:
default="Anonymous" is a good example.

I don't want primary to be confused with primary keys in sql.

Cc @wittjosiah: can we follow the same default= pattern for text?